README.md

```diff
@@ -618,22 +623,25 @@ Basic usage: `amclient.py <subcommand> [optional arguments]
 
 * aips
   * purpose: print all AIPs in the Storage Service
-  * positional argument: ss_api_key - Storage Service API key
+  * positional argument:
+    * ss_api_key - Storage Service API key
   * optional arguments:
     * `--ss-user-name <username>` - Storage Service username (default: `test`)
     * `--ss-url <url>` - Storage Service URL (default: `http://127.0.0.1:8000`)
 
 * dips
   * purpose: print all DIPs in the Storage Service
-  * positional argument: ss_api_key - Storage Service API key
+  * positional argument:
+    * ss_api_key - Storage Service API key
   * optional arguments:
     * `--ss-user-name <username>` - Storage Service username (default: `test`)
     * `--ss-url <url>` - Storage Service URL (default: `http://127.0.0.1:8000`)
 
 * aips2dips
   * purpose: print all AIPs in the Storage Service along with their
   corresponding DIPs
-  * positional argument: ss_api_key - Storage Service API key
+  * positional argument:
+    * ss_api_key - Storage Service API key
```

Comment:
Maybe `positional argument:` and `optional argument:` should be `positional argument(s):` and `optional argument(s):`. What do you think?

Reply:
Logged as Issue #83.